lips/lip-6.md

```diff
@@ -0,0 +1,79 @@
+* [`add_validator_and_reconfigure`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/add_validator_and_reconfigure.move) - to add Validator Owner to the Validator Set with its current config and trigger reconfiguration
+* [`remove_validator_and_reconfigure`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/remove_validator_and_reconfigure.move) - to remove Validator Owner from the Validator Set and trigger reconfiguration
+
+The following transaction scripts can be run by the Validator Owner:
+* [`set_validator_operator`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/set_validator_operator.move) - to designate the Validator Operator
+* [`rotate_authentication_key`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/rotate_authentication_key.move) - to rotate the authentication key of the Validator Owner Account
+
+The following transaction scripts can be run by the Validator Operator:
+* [`register_validator_config`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/register_validator_config.move) - to register the Validator Config resource
+* [`set_validator_config_and_reconfigure`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/set_validator_config_and_reconfigure.move) - to change values of the Validator Config resource and update the config in the Validator Set (aborts if the Validator Owner is not in the Validator Set)
+* [`rotate_authentication_key`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/rotate_authentication_key.move) - to rotate the authentication key of the Validator Operator Account
+
+## Formal verification guarantees on the Move Implementation
+* Disciplined aborts
+* Only LibraRoot or a Validator Operator of a Validator Owner which is currently in the Validator Set can trigger reconfiguration
+* Every change to a Validator Set triggers a reconfiguration when LibraConfig::NewEpochEvent is emitted
+* Only a Validator Owner (role) can set its Validator Operator; for each Validator Owner account this permission is given only to the Validator Owner (itself)
+* Only a Validator Operator of a Validator Owner who is present in the Validator Set can update a Validator Config in the Validator Set
+* The size of the Validator Set can only change by at most one in a block
+* Multiple calls to add_validator or remove_validator in the same block are prohibited except in genesis
+* Both functions register_validator_config or set_validator_config_and_reconfigure check the validity of the consensus key and abort if an invalid key is given
+* A Validator Owner account can only appear once in a Validator Set
+* (currently) If the Validator's Operator is not in the Validator Set, the Validator Owner itself acts as its own Validator Operator
```

Comment:
I have a bad feeling about this one.  Why not make the validator owner explicitly be the validator operator, for consistency?

Reply:
Because there going to be Validator Operator accounts who will manage configs of _multiple_ Validator Owner accounts.